Dockerfile

```diff
@@ -23,7 +23,10 @@ FROM alpine:3.15
 # The following makes debugging Agora much easier on server
 # Since it's a tiny configuration file read by GDB at init, it won't affect release build
 COPY devel/dotgdbinit /root/.gdbinit
-RUN apk --no-cache add ldc-runtime llvm-libunwind libgcc libsodium libstdc++ sqlite-libs
+COPY --from=Builder /root/packages/ /root/packages/
+RUN apk --no-cache add --allow-untrusted -X /root/packages/build/ ldc-runtime=1.28.1-r0 \
+    && rm -rf /root/packages/
+RUN apk --no-cache add llvm-libunwind libgcc libsodium libstdc++ sqlite-libs
```

Comment:
Hopefully we'll get https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/29685 soon